src/frontend/src/components/AcknowledgePolicyChanges/style.ts

```diff
@@ -0,0 +1,84 @@
+import styled from "@emotion/styled";
+import CloseIcon from "@material-ui/icons/Close";
+import { fontBodyS, getColors, getIconSizes, getSpacings, Link } from "czifui";
+import IconInfo from "src/common/icons/IconInfo.svg";
+
+// Note the use of `DummyCenteringSpacer`. Explanation with it.
```

Comment:
seems like this has been satisfied and can be removed

Reply:
This was not a very clear comment, but I wanted to point out the weird flex usage and how it depended on `DummyCenteringSpacer`. I've improved the comment, should be clear now.